integrated_channels/blackboard/client.py

```diff
@@ -14,13 +15,17 @@
 
 from integrated_channels.exceptions import ClientError
 from integrated_channels.integrated_channel.client import IntegratedChannelApiClient
+from integrated_channels.utils import refresh_session_if_expired
+
+LOGGER = logging.getLogger(__name__)
 
 # TODO: Refactor candidate (duplication with canvas client)
 GRADEBOOK_PATH = '/learn/api/public/v1/courses/{course_id}/gradebook/columns'
 ENROLLMENT_PATH = '/learn/api/public/v1/courses/{course_id}/users'
 COURSE_PATH = '/learn/api/public/v1/courses'
 POST_GRADE_COLUMN_PATH = '/learn/api/public/v2/courses/{course_id}/gradebook/columns'
 POST_GRADE_PATH = '/learn/api/public/v2/courses/{course_id}/gradebook/columns/{column_id}/users/{user_id}'
+COURSE_V3_PATH = '/learn/api/public/v3/courses/{course_id}'
```

Comment:
I don't question the accuracy, this is just a question for my understanding: Why are we using v1-v3 versions? Do they really not have the functionality all in a "v3"?

Reply:
ya they are transitioning slowly it seems

Reply:
docs are a mix of versions. Seems like certain areas are farther developed than others. Unfortunately, don't think we have much options here.